keyboards/doio/kb12/kb12.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2022 DOIO
+ * Copyright 2022 DOIO2022 <https://github.com/DOIO2022>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+/* This a shortcut to help you visually see your layout.
+ *
+ * The first section contains all of the arguments representing the physical
+ * layout of the board and position of the keys.
+ *
+ * The second converts the arguments into a two-dimensional array which
+ * represents the switch matrix.
+ */
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, \
+	K10, K11, K12, K13,      \
+	K20, K21, K22, K23       \
+) { \
+	{ K00,   K01,   K02,   K03,   K04 }, \
+	{ K10,   K11,   K12,   K13,   KC_NO }, \
+	{ K20,   K21,   K22,   K23,   KC_NO }  \
+}
```

Comment:
The layout information should ideally be moved to the `json` file using the `matrix` attribute on each key. At which point this `.h` wouldn't contain any information and coulde be removed

Reply:
> Because you have failed to do as the first sentence states?
I have deleted this file ,and what can I  do next?  Thank you!